### ISSUE TITLE
Error in make for openacc_c_main

### ISSUE THREAD

**Issue by matheustavares** [jefflarkin/openacc-interoperability#1]
I'm trying to compile openacc_c_main, but when make runs this command:
`pgcc -o openacc_c_main -fast -acc -ta=nvidia:rdc -Minfo=accel saxpy_cuda.o openacc_c_main.o -Mcuda `
I get this error:
```
saxpy_cuda.o: In function "saxpy":
tmpxft_000023f5_00000000-5_saxpy_cuda.cudafe1.cpp:(.text+0xef): undefined reference for "__cudaPushCallConfiguration"
saxpy_cuda.o: In function "__device_stub__Z12saxpy_kernelifPfS_(int, float, float*, float*)":
tmpxft_000023f5_00000000-5_saxpy_cuda.cudafe1.cpp:(.text+0x24b):  undefined reference for "__cudaPopCallConfiguration"
pgacclnk: child process exit status 1: /usr/bin/ld
```

Do you have any idea of what may be happening?

My pgcc and nvcc versions:
```
pgcc 17.10-0 64-bit target on x86-64 Linux -tp haswell 
Cuda compilation tools, release 9.2, V9.2.88
```

**Comment by matheustavares** [comment]
Solved it updating PGI to 18.4 and changing `-ta=nvidia:rdc` to `-ta=tesla:cc60,rdc` (running in geforce gtx 1050).

**Comment by jefflarkin** [comment]
@MatheusBernardino sorry you had trouble. I'm glad you were able to gt it resolved.
